--- roxterm-1.16.3/configure.ac.old	2009-12-06 13:42:50.000000000 +0100
+++ roxterm-1.16.3/configure.ac	2009-12-06 13:45:23.000000000 +0100
@@ -175,9 +175,9 @@
 m4_pattern_allow(PKG_DATA_DIR)
 AC_DEFINE_UNQUOTED([PKG_DATA_DIR], "$dir/$PACKAGE", [pkgdatadir])
 m4_pattern_allow(PKG_DOC_DIR)
-AC_DEFINE_UNQUOTED([PKG_DOC_DIR], "$dir/doc/$PACKAGE",
+AC_DEFINE_UNQUOTED([PKG_DOC_DIR], "$dir/doc/$PACKAGE-${PACKAGE_VERSION}",
            [Where package's documentation is installed])
-AC_SUBST(pkgdocdir, ["$dir/doc/$PACKAGE"])
+AC_SUBST(pkgdocdir, ["$dir/doc/$PACKAGE-${PACKAGE_VERSION}"])
 m4_pattern_allow(ICON_DIR)
 pkgicondir="$dir/icons/hicolor/scalable/apps"
 AC_DEFINE_UNQUOTED([ICON_DIR], "$pkgicondir", [Icon dir])
