--- roxterm-2.4.2/maitch.py~	2012-01-04 18:03:30.000000000 +0100
+++ roxterm-2.4.2/maitch.py	2012-02-07 14:16:43.209307382 +0100
@@ -1588,7 +1588,7 @@
         set_default(kwargs, 'targets', ".lo")
         set_default(kwargs, 'rule',
                 "${LIBTOOL} --mode=compile --tag=CC ${LIBTOOL_FLAGS_} "
-                "gcc ${LIBTOOL_MODE_ARG_} ${CFLAGS_} -c -o ${TGT} ${SRC}")
+                "${CC} ${LIBTOOL_MODE_ARG_} ${CFLAGS_} -c -o ${TGT} ${SRC}")
         CRule.__init__(self, **kwargs)
 
 
@@ -1630,7 +1630,7 @@
         self.init_var(kwargs, 'libtool_mode_arg')
         set_default(kwargs, 'rule',
                 "${LIBTOOL} --mode=link ${LIBTOOL_FLAGS_} "
-                "gcc ${LIBTOOL_MODE_ARG_} "
+                "${CC} ${LIBTOOL_MODE_ARG_} "
                 "${CFLAGS_} ${LIBS_} ${LDFLAGS_} -o ${TGT} ${SRC}")
         ProgramRule.__init__(self, **kwargs)
 
--- roxterm-2.4.2/mscript.py~	2012-01-04 18:03:30.000000000 +0100
+++ roxterm-2.4.2/mscript.py	2012-02-07 14:31:06.993247782 +0100
@@ -5,7 +5,7 @@
 from maitch import *
 
 ctx = Context(PACKAGE = "roxterm", SRC_DIR = "${TOP_DIR}/src",
-        CFLAGS = "-O2 -g -Wall -I. -I${SRC_DIR} -D_GNU_SOURCE -DHAVE_CONFIG_H")
+        CFLAGS = "-I. -I${SRC_DIR} -D_GNU_SOURCE -DHAVE_CONFIG_H")
 
 
 MINILIB_SOURCES = "colourscheme.c dlg.c display.c dragrcv.c dynopts.c " \
@@ -186,15 +186,15 @@
             "${CFLAGS} ${VTE_CFLAGS}", "${LIBS} ${VTE_LIBS}")
     
     ctx.setenv('CORE_CFLAGS',
-            "${CFLAGS} ${GTK_CFLAGS} ${DBUS_CFLAGS}")
+            "${OPTFLAGS} ${CFLAGS} ${GTK_CFLAGS} ${DBUS_CFLAGS}")
     ctx.setenv('CORE_LIBS',
             "${LIBS} ${GTK_LIBS} ${DBUS_LIBS}")
     ctx.setenv('ROXTERM_CFLAGS',
-            "${CFLAGS} ${VTE_CFLAGS} ${SM_CFLAGS} ${DBUS_CFLAGS}")
+            "${OPTFLAGS} ${CFLAGS} ${VTE_CFLAGS} ${SM_CFLAGS} ${DBUS_CFLAGS}")
     ctx.setenv('ROXTERM_LIBS',
             "${LIBS} ${VTE_LIBS} ${SM_LIBS} ${DBUS_LIBS}")
     ctx.setenv('ROXTERM_CONFIG_CFLAGS',
-            "${CFLAGS} ${GTK_CFLAGS} ${DBUS_CFLAGS} ${GMODULE_CFLAGS} "
+            "${OPTFLAGS} ${CFLAGS} ${GTK_CFLAGS} ${DBUS_CFLAGS} ${GMODULE_CFLAGS} "
             "-DROXTERM_CAPPLET")
     ctx.setenv('ROXTERM_CONFIG_LIBS',
             "${LIBS} ${GTK_LIBS} ${DBUS_LIBS} ${GMODULE_LIBS}")
